third_party/ibis/ibis_oracle/datatypes.py

```diff
@@ -140,7 +140,10 @@ def sa_oracle_NCHAR(_, satype, nullable=True):
 
 @dt.dtype.register(OracleDialect_cx_oracle, sa.dialects.oracle.TIMESTAMP)
 def sa_oracle_TIMESTAMP(_, satype, nullable=True):
-    return dt.Timestamp(nullable=nullable)
+    if satype.timezone:
+        return dt.Timestamp(timezone="UTC", nullable=nullable)
```

Comment:
The time zone is attached to an individual piece of data rather than to the column so whatever we put there is wrong. The other engines that already worked with time zones before this work (e.g. PostgreSQL and SQL Server) also use a 'UTC' literal as a default.

My understanding is that DVT will attempt to convert all data to UTC before comparing it (row and column comparisons) so this is not particularly wrong.

Having said that, we are also displaying that literal across the board in schema validation to signify that a timestamp is time zone aware. We could use a different literal such as `(TZ)` instead of the time zone?

Reply:
Ah, I see this makes sense. I think 'UTC' works in this case since we do convert to UTC before comparing.